pkg/ui/react-app/src/thanos/pages/blocks/BlockDetails.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { Block } from './block';
+import styles from './blocks.module.css';
+import moment from 'moment';
+
+export interface BlockDetailsProps {
+  block: Block | undefined;
+  selectBlock: React.Dispatch<React.SetStateAction<Block | undefined>>;
+}
+
+export const BlockDetails: FC<BlockDetailsProps> = ({ block, selectBlock }) => {
+  return (
+    <div className={`${styles.blockDetails} ${block && styles.open}`}>
+      {block && (
+        <>
+          <div className={styles.detailsTop}>
+            <p>{block.ulid}</p>
```

Comment:
a `<p>` doesn't seem like the most semantic tag for this; maybe a `span` or a `h#`

Reply:
Seems like a very minor thing but let's change that and merge this? :tada:

Reply:
Yes, `span` is much more semantic.